### PR TITLE
Added page documenting useful Environment Variables

### PR DESCRIPTION
**Pull request by nathanielks**

### CODE REVIEW

File: website/source/docs/configuration/environment-variables.html.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+layout: "docs"
+page_title: "Environment Variables"
+sidebar_current: "docs-config-environment-variables"
+description: |-
+  Something on Environment Variables!
```
Comment (phinze) [inline website/source/docs/configuration/environment-variables.html.md]: I'd just say something like "Details on the environment variables that can be used to configure various aspects of Terraform"

File: website/source/docs/configuration/environment-variables.html.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+```
+export TF_LOG=
+```
+
+For more on debugging Terraform, check out the section on [Debugging](/docs/internals/debugging.html).
+
+## TF_LOG_PATH
+
+This specifies where the log should persist its output to. Note that even when `TF_LOG_PATH` is set, `TF_LOG` must be set in order for any logging to be enabled. For example, to always write the log to the directory you're currently running terraform from:
```
Comment (phinze) [inline website/source/docs/configuration/environment-variables.html.md]: Should highlight that when this is unset, terraform logs to stderr.

**Comment by phinze** [comment]
Just a few inline nits, but overall this is looking great - thank you!

**Comment by nathanielks** [comment]
Okie doke, made those changes. I'm confused on one bit though and it's showing up in how I wrote things. On http://terraform.io/docs/internals/debugging.html, it says when TF_LOG is set, it outputs on stderr. Should that read stdout?

**Comment by phinze** [comment]
No stderr is correct - after rereading I think it's pretty clear how you have it. Good to go on your side?

**Comment by nathanielks** [comment]
:+1:

**Comment by catsby** [comment]
Thanks @nathanielks !

**Comment by nathanielks** [comment]
@catsby :smile:

**Pull request merged**